dsp-role/handler/src/test/scala/dsp/role/handler/RoleHandlerSpec.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.role.handler
+
+import dsp.errors.NotFoundException
+import dsp.errors.RequestRejectedException
+import dsp.role.api.RoleRepo
+import dsp.role.domain.Role
+import dsp.role.domain.RoleTestData
+import dsp.role.domain.RoleUser
+import dsp.role.repo.impl.RoleRepoMock
+import dsp.valueobjects.Id.RoleId
+import dsp.valueobjects.Permission
+import dsp.valueobjects.Role._
+import zio.ZIO
+import zio._
+import zio.test._
+
+/**
+ * This soec is used to test [[RoleHandler]]
+ */
+object RoleHandlerSpec extends ZIOSpecDefault {
+  def spec = (getRolesTest)
+
+  private val getRolesTest = suite("getRoles")(
```

Comment:
The name of the suite is not exactly correct, it should be something more generic because you are testing not only "get" operations but also updates. I agree with Balduin though that it would be nice to have multiple suites instead of one big.

Reply:
Updated.